src/BUILD

```diff
@@ -0,0 +1,85 @@
+package(default_visibility = ["//visibility:public"])
```

Comment:
Should the contents of this file go into the top-level BUILD since it's the root of the binary crate?

Reply:
no, this file is just for building salus, this is due to a difference between cargo and bazel.

Reply:
The `BUILD` file is supposed to go in the root of the crate. That's the top level of the repository itself for the Salus binary crate. And it's consistent with the examples provided in the `rules_rust` repo: https://github.com/bazelbuild/rules_rust/tree/main/examples

Reply:
fixed